dev/TEAL_opcodes.md

```diff
@@ -680,44 +733,46 @@ bit indexing begins with low-order bits in integers. Setting bit 4 to 1 on the i
 ## balance
 
 - Opcode: 0x60
-- Pops: *... stack*, uint64
+- Pops: *... stack*, any
 - Pushes: uint64
-- get balance for the requested account specified by Txn.Accounts[A] in microalgos. A is specified as an account index in the Accounts field of the ApplicationCall transaction, zero index means the sender. The balance is observed after the effects of previous transactions in the group, and after the fee for the current transaction is deducted.
+- get balance account A, in microalgos. The balance is observed after the effects of previous transactions in the group, and after the fee for the current transaction is deducted.
 - LogicSigVersion >= 2
 - Mode: Application
 
+params: Txn.Accounts offset (or, since v4, an account address that appears in Txn.Accounts or is Txn.Sender). Return: value.
```

Comment:
I like your wording, but I'll wait for a final consensus with Shai, since there are a bunch to change.

Reply:
I like Fabrice's wording

Reply:
This worked well for opcodes with a single parameter (balance, min_balance, asset_params_get). However, for opcodes that take multiple parameters, the repetition of all parameters made them hard to follow. The parenthetical "or" clauses seemed to work better, as they kept the parameters in order.  Since I believe both are correct, I'd like to leave those alone. If anything, perhaps saying these things two different ways for the different opcodes might make it clearer.  Maybe?  What do you think?